packages/blade/rollup.config.js

```diff
@@ -61,13 +62,17 @@ const getWebConfig = ({ exportCategory }) => ({
   input: `${inputRootDirectory}/${exportCategory}/index.ts`,
   output: [
     {
-      file: `${outputRootDirectory}/${exportCategory}/index.web.js`,
+      file: `${outputRootDirectory}/${exportCategory}/index.${process.env.NODE_ENV}.web.js`,
```

Comment:
Lets have a check  in rollup to ensure NODE_ENV is set? Otherwise it might end up creating `index.undefined.web.js` 😆

Reply:
I've passed `NODE_ENV` in our build scripts. Ideally, we should fallback to development:
```suggestion
      file: `${outputRootDirectory}/${exportCategory}/index.${process.env.NODE_ENV || 'development'}.web.js`,
```